tilelang/intrinsics/mma_macro_generator.py

```
@@ -500,24 +500,19 @@ def _warp_ldmatrix_a(
-                # Assign A_shared_buf_elem
                 wi, wk = warp_m * warp_row_tiles + i * micro_size_x, rk * chunk + ki * micro_size_k
-                A_shared_buf_elem = (
-                    A_buf[tuple(A_other) + (A_base0 + wk, A_base1 + wi)]
-                    if a_transposed
-                    else A_buf[tuple(A_other) + (A_base0 + wi, A_base1 + wk)]
-                )
 
                 if ldmatrix_available:
+                    row_off, col_off = get_ldmatrix_offset("A", tx, 0, stride, a_dtype, a_transposed)
+                    if a_transposed:
+                        src_indices = tuple(A_other) + (A_base0 + wk + row_off, A_base1 + wi + col_off)
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)